lib/tasks/load_us_counties.rake

```diff
@@ -0,0 +1,16 @@
+require 'csv'
+namespace :db do
+  desc "Load US Counties"
+  task :load_us_counties => :environment do
+    if County.find_by(name:"Abbeville County", region: " South Carolina").nil?
```

Comment:
This seems kind of arbitrary - should we just do `if County.none?`?

Reply:
Yeah... A situation where you need US counties and there are already counties, but none of them are US is rather unlikely [I can imagine it, because this is open source, but it's a stretch].  In next push.